Headers/DebugServer2/Host/Windows/ExtraWrappers.h

```diff
@@ -31,6 +31,13 @@ extern "C" {
 
 #define UNLEN 256
 
+WINBASEAPI SIZE_T WINAPI VirtualQueryEx(
```

Comment:
Not a functional change, but can you put this next to `VirtualAllocEx` and friends?